docs/distributions/nixos/installation.md

```diff
@@ -185,15 +209,74 @@ If you would like to organize your configuration a little better, check out othe
 
 ## Wi-Fi and Bluetooth setup
 
+### Pre-Installation Steps
+
+Choose a method below and carefully read the notes. Afterwards, follow the [Wi-Fi and Bluetooth Guide on macOS](../../guides/wifi-bluetooth.md#setting-up) (firmware guide) and come back.
```

Comment:
Probably mention that the methods mentioned below correspond to the ones in the wifi guide.